src/main/scala/edu/ie3/simona/ontology/StopMessage.scala

```diff
@@ -0,0 +1,9 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.ontology
```

Comment:
Why not `edu.ie3.simona.ontology.message`?

Reply:
Ok, will do